.github/dependabot.yml

```diff
@@ -18,84 +18,82 @@ updates:
-      - dependency-name: "@types/echarts"
-      - dependency-name: "@types/marked"
-      - dependency-name: "@types/react-router"
-      - dependency-name: "@types/react-select"
-      - dependency-name: "@types/reflux"
-      - dependency-name: "@visual-snapshot/jest"
-      - dependency-name: "@wojtekmaj/enzyme-adapter-react-17"
-      - dependency-name: "babel-jest"
-      - dependency-name: "bootstrap"
-      - dependency-name: "clean-webpack-plugin"
-      - dependency-name: "compression-webpack-plugin"
-      - dependency-name: "copy-text-to-clipboard"
-      - dependency-name: "create-react-class"
-      - dependency-name: "dotenv-webpack"
-      - dependency-name: "echarts"
-      - dependency-name: "echarts-for-react"
-      - dependency-name: "eslint"
-      - dependency-name: "eslint-plugin-simple-import-sort"
-      - dependency-name: "fuse.js"
-      - dependency-name: "gettext-parser"
-      - dependency-name: "jest"
-      - dependency-name: "jest-junit"
-      - dependency-name: "less"
-      - dependency-name: "marked"
-      - dependency-name: "mini-css-extract-plugin"
-      - dependency-name: "react-lazyload"
-      - dependency-name: "react-popper"
-      - dependency-name: "react-refresh"
-      - dependency-name: "react-router"
-      - dependency-name: "react-select"
-      - dependency-name: "reflux"
-      - dependency-name: "rrweb"
-      - dependency-name: "rrweb-player"
-      - dependency-name: "sprintf-js"
-      - dependency-name: "u2f-api"
+      - dependency-name: '@sentry/release-parser'
+      - dependency-name: '@types/echarts'
+      - dependency-name: '@types/marked'
+      - dependency-name: '@types/react-router'
+      - dependency-name: '@types/react-select'
+      - dependency-name: '@types/reflux'
+      - dependency-name: '@visual-snapshot/jest'
+      - dependency-name: '@wojtekmaj/enzyme-adapter-react-17'
+      - dependency-name: 'babel-jest'
+      - dependency-name: 'bootstrap'
+      - dependency-name: 'clean-webpack-plugin'
+      - dependency-name: 'compression-webpack-plugin'
+      - dependency-name: 'copy-text-to-clipboard'
+      - dependency-name: 'create-react-class'
+      - dependency-name: 'dotenv-webpack'
+      - dependency-name: 'echarts'
+      - dependency-name: 'echarts-for-react'
+      - dependency-name: 'fuse.js'
+      - dependency-name: 'gettext-parser'
+      - dependency-name: 'jest'
+      - dependency-name: 'jest-junit'
+      - dependency-name: 'less'
+      - dependency-name: 'marked'
+      - dependency-name: 'mini-css-extract-plugin'
+      - dependency-name: 'react-lazyload'
+      - dependency-name: 'react-popper'
+      - dependency-name: 'react-refresh'
+      - dependency-name: 'react-router'
+      - dependency-name: 'react-select'
+      - dependency-name: 'reflux'
+      - dependency-name: 'rrweb'
+      - dependency-name: 'rrweb-player'
+      - dependency-name: 'sprintf-js'
+      - dependency-name: 'u2f-api'
```

Comment:
Annoyingly prettier formatted this. The main change is that I removed eslint from the ignored list in here